openlibrary/plugins/openlibrary/js/jquery.repeat.js

```diff
@@ -39,25 +39,25 @@ export default function($){
         /**
          * Search elems.form for input fields and create an
          * object representing.
-         * This function has side effects and will reset any
-         * input[type=text] fields it has found in the process
          * @return {object} data mapping names to values
          */
         function formdata() {
             var data = {};
             $(':input', elems.form).each(function() {
                 var $e = $(this),
+                    name = $e.attr('name'),
                     type = $e.attr('type'),
-                    name = $e.attr('name');
+                    id = $e.attr('id');
```

Comment:
```suggestion
                    _id = $e.attr('id');
```
There's another globally scoped variable named `id` (defined in this file).  Changing this to avoid accidental reassignments.

Reply:
Makes sense and looks great! Thanks for catching!